client/src/main/scala/com.gu.contentapi.client/utils/CapiModelEnrichment.scala

```diff
@@ -105,6 +105,7 @@ object CapiModelEnrichment {
         isPictureContent -> PictureDesign,
         tagExistsWithId("type/audio") -> AudioDesign,
         tagExistsWithId("type/video") -> VideoDesign,
+        tagExistsWithId("type/crosswords") -> CrosswordDesign,
```

Comment:
Should this be `type/crossword`? It looks to be this in the [Example](https://www.theguardian.com/crosswords/quick/17058.json?dcr) provided in the PR description.

Reply:
Indeed it should, thank you!